setup.py

```diff
@@ -1,6 +1,9 @@
+from __future__ import with_statement
+from __future__ import absolute_import
 from codecs import open
 from os import path
 from setuptools import setup, find_packages
+from io import open
```

Comment:
This overwrites the previous import of `open`. Python 2.7 supports `codecs.open` anyway, right?